datadog_lambda/wrapper.py

```diff
@@ -201,10 +205,17 @@ def _after(self, event, context):
                 if status_code:
                     self.span.set_tag("http.status_code", status_code)
                 self.span.finish()
+
             if self.inferred_span:
+                # TODO FIXME: should we move this logic to
+                # the inferred span methods where this is relevant?
```

Comment:
I think we don't know the status code until `_after`, and we don't know `self.span.start` until after the inferred span has been created. We could break this out into a `finish_inferred_span` method if you think that would be cleaner

Reply:
Okay yes, you're right we don't have the status code until `_after` (that's what this comment was referring to). The rest looks fine, I'm gonna go through and refactor the entire inferred spans logic sometime soon so I'll move this bit into a `_ finish_inferred_span` method.